### DISCUSSION TITLE
Error when running brats segmentation model (v 0.3.8) from monaibundle

### DISCUSSION THREAD

**Discussion by tdakinosho** [Project-MONAI/MONAILabel#1305] [Q&A]
Hello community,

I am currently having troubles running the brats model for brain segmentation on 3D slicer. I have a custom dataset(originally DICOM images but converted to NIFTI using SimpleiTK library) that i am initially trying to run the model on before i commence training but i am stuck with this. Everything works fine when i tried using  deepedit from the radiology app but since my dataset is of brain scans, i would rather have the brats model work fine. I have tried going through the suggestions on this discussion thread (https://github.com/Project-MONAI/MONAILabel/issues/1051) but i could not find a solution to my error. Please find the error i am experiencing below and what i have tried so far.

Error:
**RuntimeError: Error(s) in loading state_dict for SegResNet:
	size mismatch for convInit.conv.weight: copying a param with shape torch.Size([16, 4, 3, 3, 3]) from checkpoint, the shape in current model is torch.Size([16, 1, 3, 3, 3]).**

What i have tried so far:
Updated the inference.json in monaibundle\brats_mri_segmentation_v0.3.8\configs with 'ensure_channel_first' arg
Changed the "in_channels" value to 1 from 4 in brats_mri_segmentation_v0.3.8/configs/train.json ,inference.json and metadata.json.
I have tried downloading and using the 'Task01_BrainTumour' dataset but i got the same error

Please any help or suggestions would be appreciated

**Comment by diazandr3s** [comment]
Hi @tdakinosho,

Sorry I completely missed your question.

Can I ask how many modalities are you using for this? All four as in the BRATS dataset? Have you tried using the MONAI [Auto3DSeg module in Slicer](https://github.com/lassoan/SlicerMONAIAuto3DSeg/)?

Just download the latest Slicer and then install Auto3DSeg via the Extension Manager

Please let us know,
